<launch>
  <node pkg="car_I2C_control" type="car_I2C_control_node" name="car_I2C_control_node" output="screen">
		<param name = "speed_factor"   type = "double"    value  = "255" />
		<param name = "steer_factor"  type = "double"     value  = "20"/>

 </node> 
  
  
  
</launch>
